MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMovies-ViewController.swift

```diff
@@ -6,56 +6,62 @@
 //
 
 import UIKit
-import SDWebImage
 
 class SearchMoviesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
-    
-    
+
     private let searchMoviesViewModel = SearchMoviesViewModel()
-    
-    
-    @IBOutlet weak var searchMovieTableView: UITableView!
-    
+
+    @IBOutlet private weak var searchMovieTableView: UITableView!
     override func viewDidLoad() {
         super.viewDidLoad()
-        let nib = UINib(nibName: "SearchMovieTableViewCell", bundle: nil)
-        searchMovieTableView.register(nib, forCellReuseIdentifier: "reuseCell")
+        let nib = UINib(nibName: Constants.Identifiers.nibIdentifier, bundle: nil)
+        searchMovieTableView.register(nib, forCellReuseIdentifier: Constants.Identifiers.tableCellReuseIdentifier)
```

Comment:
This can be moved into a function called setUpTableView and then call the function here for clarity. Look at Kaits/Gia's PR😄